CHANGELOG.rst

```diff
@@ -2,14 +2,192 @@
+
+* Added documentation on how to configure Camunda for DMN.
+* Tweaked the dark-mode styling of WYSIWYG editors to better fit in the page.
+* [#3164] Added explicit timeout fields to services so they can be different from the
+  global default.
+* [#3695] Improved login screen and flow
+
+    - Allow opt-in to automatically redirect to OIDC provider.
+    - Support WebAuthn (like YubiKey) hardware tokens.
+
+* [#3885] The admin form list now keeps track of open/collapsed form categories.
+* [#3957] Updated the eIDAS logo.
+* [#3825] Added a well-performing public API endpoint to list available forms, returning
+  only minimal information.
+* [#3825] Added public API endpoint to list available form categories.
+* [#3879] Added documentation on how to add services for the service fetch feature.
+* [#3823] Added more extensive documentation for template filters, field regex validation
+  and integrated this documentation more into the form builder.
+* [#3950] Added additional values to the eHerkenning CSP-header configuration.
+* [#3977] Added additional validation checks on submission completion of the configured
+  formio components in form steps.
+* [#4000] Deleted the 'save and add another' button in the form designer to maintain safe
+  blood pressure levels for users who accidentally clicked it.
+
+**Bugfixes**
+
+* [#3672] Fixed the handling of object/array variable types in service fetch configuration.
+* [#3890] Fixed visually hidden fields not being sent to Objects API registration backend.
+* [#1052] Upgraded DigiD/eHerkenning library.
+* [#3924] Fixed updating of payment status when the "registration after payment is
+  received" option is enabled.
+* [#3909] Fixed a crash in the form designer when removing form variables that are used
+  in the case properties mapping.
+* [#3921] Fixed not all (parent/sibling) components being available for selection in the
+  form builder.
+* [#3922] Fixed a crash because of invalid prefill configuration in the form builder.
+* [#3958] Fixed the preview appearance of read-only components.
+* [#3961] Reverted the merged KVK API services (basisprofiel, zoeken) back into separate
+  configuration fields. API gateways can expose these services on different endpoints.
+* [#3705] Fixed the representation of timestamps (again).
+* [#3975,#3052] Fixed legacy service fetch configuration being picked over the intended
+  format.
+* [#3881] Fixed updating a re-usable form definition in one form causing issues in other
+  forms that also use this same form definition.
+* [#4022] Fix crash on registration handling of post-payment registration. The patch for
+  #3924 was bugged.
+* [#2827] Worked around an infinite loop when assigning the variable ``now`` to a field
+  via logic.
+* [#2828] Fixed a crash when assigning the variable ``today`` to a variable via logic.
+
+**Project maintenance**
+
+* Removed the legacy translation handling obsoleted by the new form builder.
```

Comment:
Is "to obsolate" a verb? :thinking: I thought it was "to make obsolete"?

Reply:
looked it up, it's not :) I feex